cekit/template_helper.py

```diff
@@ -87,3 +87,10 @@ def ports(self, available_ports):
     def cachito(self, image):
         if image.get('osbs', {}).get('configuration', {}).get('container', {}).get('remote_source'):
             return True
+
+    def default_package_manager_flags(self, pkg_mgr):
+        default = "--setopt=tsflags=nodocs"
```

Comment:
I would go one step further. Since we allow to use multiple package managers, some of them will not accept this default value (`apk` as an example). I would return empty string in case of apk, `--setopt=tsflags=nodocs` for other supported ones and add `--setopt=install_weak_deps=0`

Other thing is I don't know how did it work earlier with apk :)

Reply:
@goldmann Perhaps apk never worked! I imagine none of the tests cover it

Reply:
Hm, we have a few tests. Needs to be checked.

Reply:
@goldmann The `apk` is handled within the `template.jinja` by a separate 'if path' so that it doesn't actually use default_package_manager flags. This is why it worked before (and still does now)

Reply:
How about generalizing it into "install" and "verify" methods in helper depending on the package manager used? In other words - move code from template into helper.

Reply:
This would help with maintenance of the template and makes it handled in single place.

Reply:
Updated now